### PR TITLE
Automatically formats dart code.

### PR DESCRIPTION
**Pull request by digiter**
Formats all dart code with `dartfmt`, except app_flutter which has already been formatted with flutter formatter.

### CODE REVIEW

**Comment by CaseyHillers** [comment]
Thanks for doing this Tong! If you rebase with master `app` should start passing.

**Comment by digiter** [comment]
Thanks for the reminder, rebasing to master worked!

File: .cirrus.yml [hunk 1]
```diff
@@ -10,9 +10,10 @@ task:
       - CWD: agent
   install_tuneup_script: pub global activate tuneup
   pub_script: cd $CWD && pub get
+  format_script: cd $CWD && dartfmt --set-exit-if-changed .
```
[the remark this answers is not in the document]
Comment (digiter) [reply .cirrus.yml]: This formatting has included `*_test.dart`, or do you mean putting this line into `./test.sh`?

[the remark this answers is not in the document]
Comment (godofredoc) [reply .cirrus.yml]: Actually I was thinking on https://github.com/flutter/cocoon/blob/master/test_utilities/bin/dart_test_runner.sh

[the remark this answers is not in the document]
Comment (digiter) [reply .cirrus.yml]: Done.

**Pull request merged**